components/sidebar/sidebar_menu/sidebar_menu.tsx

```diff
@@ -59,22 +59,6 @@ export default class SidebarMenu extends React.PureComponent<Props, State> {
         }
     }
 
-    // TODO: Temporary code to keep the menu in place while scrolling
-    // This shouldn't be necessary once the menus are fixed up
-    componentDidMount() {
```

Comment:
Are these just not needed anymore?

Reply:
This is breaking the functionality that closes the menus when you scroll the sidebar, but I couldn't come up with a quick, low-risk way to do that with how the menus are written. I think the performance impacts of this change heavily outweigh the slight UX degredation.

Reply:
Can you file a ticket so this can be revisited?